eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -76,14 +76,14 @@ library
         cardano-crypto-class,
         cardano-crypto-test,
         cardano-crypto-wrapper,
-        cardano-data,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.2,
+        cardano-data ^>=1.2,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib} ^>=1.3,
```

We try not to go too crazy on speculative upper bounds for these packages, cause they cause redundant work for release process.
```suggestion
        cardano-data >=1.2,
        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.3,
```

For example right now even bumping the lower bound is not necessary, since it will be enforce transitively through `cardano-ledger-shelley` and other packages.

Technically the only lower bound that is needed changing is:

```
cardano-ledger-shelley:{cardano-ledger-shelley, testlib} ^>=1.9,
```

Cause that is the only change in the code that is necessary for `cardano-ledger-shelley-test`. It won't hurt, but it is not necessary.